clusterloader2/cmd/clusterloader.go

```diff
@@ -80,6 +81,7 @@ func validateClusterFlags() *errors.ErrorList {
 func initFlags() {
 	flags.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which cause reports being written to standard output.")
 	flags.BoolEnvVar(&clusterLoaderConfig.EnablePrometheusServer, "enable-prometheus-server", "ENABLE_PROMETHEUS_SERVER", false, "Whether to set-up the prometheus server in the cluster.")
+	flags.BoolEnvVar(&clusterLoaderConfig.EnableExecService, "enable-exec-service", "ENABLE_EXEC_SERVICE", false, "Whether to enable exec service for the measurements.")
```

Comment:
"for the measurements" - this is an implementation detail that this service is going to be used from measurements, but doesn't explain the purpose of the exec service. Could you rewrite? Something like, ... exec service that allows executing arbitrary commands from a pod running in the cluster, e.g. calling curl on an IP address to check connectivity.

Reply:
Done.